bundle/manifests/lightspeed-operator.clusterserviceversion.yaml

```diff
@@ -562,8 +578,8 @@ spec:
   version: 0.0.1
   relatedImages:
     - name: lightspeed-service-api
-      image: quay.io/redhat-user-workloads/crt-nshift-lightspeed-tenant/ols/lightspeed-service@sha256:b8e0bfe7e4a241885bcd30baa7066239276d5a6c7de8b6526d4861c6d3d4bfe7
+      image: quay.io/openshift-lightspeed/lightspeed-service-api:latest
     - name: lightspeed-console-plugin
-      image: quay.io/redhat-user-workloads/crt-nshift-lightspeed-tenant/ols/lightspeed-console@sha256:069828c27813e618dda86a498946d06d27daf73a8ff8d601a3cb72d00324a900
+      image: quay.io/openshift-lightspeed/lightspeed-console-plugin:latest
```

Comment:
need to set them back to konflux images manually.
I will update the script later.